randovania/games/am2r/logic_database/Genetics Laboratory.txt

```diff
@@ -715,6 +718,10 @@ Extra - minimap_data: [{'x': 7, 'y': 20}, {'x': 8, 'y': 20}, {'x': 9, 'y': 20}]
                   Combat (Intermediate) and Normal Damage ≥ 400
                   Combat (Advanced) and Normal Damage ≥ 300
                   Combat (Expert) and Normal Damage ≥ 200
+              # Dodge
+              Hi-Jump Boots or Space Jump or Walljump (Advanced)
+              # Dodge goo floor
+              Space Jump or Walljump (Expert) or Can Use Spider Ball
```

Comment:
i feel like this wall jump should be bumped down to advanced.
and yeah, it should also be 
ANY
- spider
- ALL
  - ANY
    space jump or WJ
  - ANY
    PBs or screw attack

Reply:
I think this needs to be rearranged to (or similar):
- ALL
    - ANY
          - Spider or space or WJ
    - ANY
          - PBs or Screw

Reply:
that doesnt make as much sense
if you have spider, you can just camp in the upper right corner in that phase, and no blob will ever hit you

Even if it would be able to hit you, spider + screw doesnt make sense either.

Reply:
Didn't realize you could safe spot the blobs in spider..